anitya/lib/backends/custom.py

```diff
@@ -31,6 +31,7 @@ class CustomBackend(BaseBackend):
     more_info = 'More information in the '\
         '<a href=\'/about#test-your-regex\'>about#test-your-regex</a>'
     default_regex = REGEX % {'name': '{project name}'}
+    default_regex_html = default_regex.replace('\\', '\\\\')
```

Comment:
Since this is only relevant in the HTML template, I'd recommend just putting it there. Jinja lets you write Python in the templates.